baseui/templates/baseui/base.html

```diff
@@ -36,6 +36,16 @@
     </div>
     <!-- contains arXiv identity and search bar -->
     <div class="identity"><h1><a href="">arXiv.org</a></h1></div>
+    <div class="user-tools">Login | My Account | Logout</div>
+    <div class="search-block">
+      <form>
```

Comment:
I hate to advocate hard-coding anything, but the only thing missing here is ``method="POST" action="https://arxiv.org/search"`` ; @mhl10 what do you think?